src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -178,8 +178,7 @@
 {% block form_help -%}
     {%- if help is not empty -%}
         {%- set help_attr = help_attr|merge({class: (help_attr.class|default('') ~ ' help-block')|trim}) -%}
-
-        <span id="{{ id }}_help" {% for attrname, attrvalue in help_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
+        <span id="{{ id }}_help"{% for attrname, attrvalue in help_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
```

Comment:
actually, for compatibility shouldnt we use `{% with { attr: help_attr } %}{{ block('attributes') }}{% endwith %}`

Reply:
this is indeed missing some of the features of the `attributes` block

Reply:
Good idea, added them in the commit.